src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/SyntaxTreeExtensions.cs

```diff
@@ -110,13 +110,19 @@ public static bool IsInNonUserCode(this SyntaxTree syntaxTree, int position, Can
                             break;
                         }
 
-                    case SyntaxKind.GreaterThanGreaterThanToken:
-                        stack++;
-                        goto case SyntaxKind.GreaterThanToken;
```

Comment:
FWIW, I did a test CI run seeing which tests had a >> token _and_ returned true from this function -- we didn't have any until the tests I added it seems.